src/app/components/hris/charts/charts.component.ts

```diff
@@ -59,6 +60,8 @@ export class ChartComponent implements OnInit {
   selectedChartIndex: number = -1;
   barChartOptions = barChartOptions;
   pieChartOptions = pieChartOptions;
+  dialogTypeData: Dialog = { type: '', title: '', subtitle: '', confirmButtonText: '', denyButtonText: '' };
```

Comment:
Is this object declaration being repeated like this throughout the project?

Reply:
![image](https://github.com/RetroRabbit/RGO-Client/assets/82169901/d6d149a8-cec9-4d52-bdd6-c30ec8de6091)